.github/workflows/zowe-versions_updateVersions_createPR.yml

```diff
@@ -73,7 +73,8 @@ jobs:
           branch_name="v$version_name/master"
           pr_title="Update zowe component versions for $version_name"
           body_file=description.txt
-          pr_number=$(gh pr create --base master --head "$branch_name" --title "$pr_title" --body-file "$body_file") | sed 's/.*\/\([0-9]\+\)$/\1/'
+          pr_url=$(gh pr create --base master --head "$branch_name" --title "$pr_title" --body-file "$body_file")
+          pr_number=$(echo "$pr_url" | sed 's/.*\/\([0-9]\+\)$/\1/')
           curl -X POST \
             -H "Accept: application/vnd.github+json" \
             -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}"\

```

Comment:
I see that we are adding a few people IDs as reviewiers...

Could we just have the @zowe/zowe-cli-administrators team as the reviewier?

Reply:
I think we could since the cURL command that we based on supports passing `team_reviewers` in the payload too 😋 
https://github.com/cli/cli/issues/4844#issuecomment-1534825316

Reply:
Good idea @zFernand0! Thanks for the reference @t1m0thyj
I've made the change 👍